### ISSUE TITLE
Change revision repetition counts

### ISSUE THREAD

**Issue by kagetora0924** [HaolinZhong/PMCA#16]
I think now it defaults to solving each problem 5 times. For certain problems, like important ones, it would be nice to have an option to review them 6,7,8,or even more times.

**Comment by HaolinZhong** [comment]
Yes, in the future I'd support this feature, although for now it is a low priority one though
